pom.xml

```diff
@@ -352,98 +359,33 @@
                             <!-- Trello card for this exclude: https://trello.com/c/ahkwEM74/1720-babbage-elasticsearch-library-multiple-vulnerabilities-s2 -->
                             <groupId>org.elasticsearch</groupId>
                             <artifactId>elasticsearch</artifactId>
-                            <version>2.1.1</version>
+                            <version>2.4.6</version>
                         </exlude>
                         <exclude>
-                            <!-- This is a transitive dependency of org.elasticsearch:elasticsearch:2.1.1 -->
+                            <!-- This is a transitive dependency of org.elasticsearch:elasticsearch:2.4.6 -->
                             <groupId>org.apache.lucene</groupId>
                             <artifactId>lucene-queryparser</artifactId>
-                            <version>5.3.1</version>
+                            <version>5.5.4</version>
                         </exclude>
-
                         <exlude>
                             <!-- Trello card for this exclude: https://trello.com/c/MCpbYlIT/1721-babbage-netty-multiple-vulnerabilities-s1 -->
                             <!-- This is a transitive dependency of org.elasticsearch:elasticsearch:2.1.1 -->
```

Comment:
2.4.6?

Reply:
because we have a dependency of elasticsearch 2.4.6 we want to exclude them here.

Reply:
👍